internal/mocks/extradatadecoder.go

```diff
@@ -0,0 +1,19 @@
+package mocks
```

Comment:
Use [.mockery.yml](https://github.com/smartcontractkit/chainlink-ccip/blob/main/.mockery.yaml) to generate a mock.